frontend/src/lib/ConnectionManager.ts

```diff
@@ -164,8 +172,9 @@ export class ConnectionManager {
       onMessage: this.props.onMessage,
       onConnectionStateChange: this.setConnectionState,
       onRetry: this.showRetryError,
-      getHostAuthToken: this.props.getHostAuthToken,
-      setHostAllowedOrigins: this.props.setHostAllowedOrigins,
+      claimHostAuthToken: this.props.claimHostAuthToken,
+      resetHostAuthToken: this.props.resetHostAuthToken,
+      setAllowedOriginsResp: this.props.setAllowedOriginsResp,
```

Comment:
It seems like ConnectionManager & WebsocketConnection should be merged or refactored because we are passing so much stuff downward. I guess it's not bad, but it just convoluted.

Reply:
Possibly -- these two classes were historically separate because we've had connections to servers other than websocket connections. I've heard some discussion that may end up leading us to having different connection types once again, so it'd likely make sense to hold off on considering a refactor for this unless maintaining this code gets painful and we don't end up adding more connection types.